From 7affe047956a0f1c604cbe2a126bd0aa25b5e1c4 Mon Sep 17 00:00:00 2001
From: Jani Hautakangas <jani.hautakangas@lge.com>
Date: Mon, 15 Jan 2018 16:08:04 +0200
Subject: [PATCH 1/5] Change conflicting mojo enum value

must.mojom.KeyboardCode.KEY_DELETE conflicts
with KEY_DELETE macro in /usr/include/linux/input-event-codes.h:
- mus.mojom.KeyboardCode.KEY_DELETE became DELETE_KEY
---
 ui/events/mojo/keyboard_codes.mojom | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/events/mojo/keyboard_codes.mojom b/ui/events/mojo/keyboard_codes.mojom
index ccb55ed3a248..c46e78fba3e4 100644
--- a/ui/events/mojo/keyboard_codes.mojom
+++ b/ui/events/mojo/keyboard_codes.mojom
@@ -44,7 +44,7 @@ enum KeyboardCode {
   EXECUTE                   = 0x2B,
   SNAPSHOT                  = 0x2C,
   INSERT                    = 0x2D,
-  KEY_DELETE                = 0x2E,
+  DELETE_KEY                = 0x2E,
   HELP                      = 0x2F,
   NUM_0                     = 0x30,
   NUM_1                     = 0x31,
-- 
2.11.0

